presto-spi/src/main/java/com/facebook/presto/spi/AriaFlags.java

```diff
@@ -24,4 +24,5 @@ private AriaFlags() {}
     public static final int noReaderBudget = 16;
     public static final int exchangeReusePages = 32;
     public static final int pruneSubfields = 64;
+    public static final int budgetByException = 128;
```

Comment:
This flag is an implementation detail of the Hive connector, hence, let's replace it with a Hive session property.